root/static/stylesheets/ng/faceted-search.css

```diff
@@ -54,6 +54,13 @@ faceted-search .search-results {
   margin-top: 2em;
 }
 
+faceted-search .search-results-horizontal {
```

Comment:
No need to include no-op CSS, unless this is somehow preventing some other style from being applied due to the cascade and I'm not noticing.

Reply:
Yeah I put that in as a stub for adjusting stuff but it turned out not to be necessary and I forgot to remove it.